ui/desktop/app/components/dialog-detail/index.hbs

```diff
@@ -0,0 +1,117 @@
+<Rose::Dialog
+  class='connection-dialog'
+  @heading={{@heading}}
+  @dismiss={{@dismiss}}
+  @dismissButtonText={{@dismissButtonText}}
+  @style={{@style}}
+  @icon={{@icon}}
+  @modal={{true}}
+  as |dialog|
+>
+  {{yield (hash
+    notification=(component 'rose/notification' class='connection-dialog-notification')
+  )}}
+
+  <dialog.body>
+    {{yield (hash
+      body=(component 'rose/anonymous')
+    )}}
+    {{!--
```

Comment:
Please, ignore this commented code, I am just leaving it temporary for reference.